portfolio/src/main/webapp/index.html

```diff
@@ -2,16 +2,24 @@
 <html>
   <head>
     <meta charset="UTF-8">
+    <img src="/Images/Name.png">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1 style="background-color:Violet;">
+        About Me: 
+        <a href="https://www.github.com/Sami-2000">Github</a>
+        <a href="https://www.linkedin.com/in/samantha-alves-865093195/">LinkedIn</a>
+      </h1>
+      <img src="/Images/Trapeze.jpg">
+      <p>Hi all! My name is Sami and I am a Junior at Yale University studying Physics and Computer Science. I love both exploring and codifying the patterns that govern our world.</p>
+      <p>When I am not studying, I work in an interdisciplinary Neuroscience lab studying fly behavior, give tours to perspective students, and tutor students in an introductory Physics class.</p>
+      <p>I also love practicing on the Club Gymnastics team, planning events for the Women in Physics group, volunteering in local elementary schools, and debating in the Yale Political Union. </p>
```

Comment:
You should add line breaks within this text so that everything fits on the pull request review screen without having to scroll. (It'll also be easier for you to read because you won't have to scroll either). Line breaks within <p> tags get ignored, so it won't affect how this text is displayed on your website, but it will make your code easier to read.